src/ui/map.js

```diff
@@ -1038,6 +1038,17 @@ class Map extends Camera {
         return {name: "mercator", center:[0, 0]};
     }
 
+    /**
+     * Returns true if map [projection](https://docs.mapbox.com/mapbox-gl-js/style-spec/projection/) has been set to globe AND the map is at a low enough zoom level that globe view is enabled.
+     *
+     * @returns {boolean} Returns `globe-is-active` boolean.
+     * @example
+     * if (map.isGlobe()) {
+     *     // do globe things here
+     * }
+     */
+    usingGlobe(): boolean { return this.transform.projection.name === 'globe'; }
```

Comment:
I'm not in favor of adding a purely globe specific helper in the public API. From the marker code it's redundant to directly call `this.transform.projection.name === 'globe'`.

If we ever need to expose that to the user, we could instead extend the current API instead of a new one, such has `map.getProjection(explicit?)`.

Reply:
My thought was that since checking for globe view is a widely useful feature it would simplify code (and potential future maintenance) to call a function directly from `map` instead of the long-winded `this.map.transform.projection.name === 'globe'`.

My guess is that users would also benefit from this, but I'm less certain about that, and I agree that we shouldn't expand the API unnecessarily. What would you think about a private function `_usingGlobe`?

Reply:
Private functions like `_usingGlobe` are fine for these type of helpers, yes! As long as we don't commit to and add extraneous public APIs unless we really need to.